### PR TITLE
Function for ETL job

### PR DESCRIPTION
**Pull request by Xinyihe123**
Fixed: #8 

The file transfer.py:
    Given an organization ID, find the target organization, related planters, trees, and species and insert them into the target database.
    Need to check the mentioned table names and column names are correct.

### CODE REVIEW

File: python/ETL/transfer.py [hunk 1]
```diff
@@ -0,0 +1,160 @@
+#DATABASE_URL=postgresql://doadmin:l5al4hwte8qmj6x8@db-postgresql-sfo2-nextgen-do-user-1067699-0.db.ondigitalocean.com:25060/treetracker?ssl=true
+
+import psycopg2
+import argparse
+import datetime
+
+DATABASE_URL="postgresql://doadmin:l5al4hwte8qmj6x8@db-postgresql-sfo2-nextgen-do-user-1067699-0.db.ondigitalocean.com:25060/treetracker?ssl=true"
```
Comment (dadiorchen) [inline python/ETL/transfer.py:7]: Please don't paste database_url here, it will compromise our db.

File: python/ETL/transfer.py [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Transfer data from source to target database.")
+    parser.add_argument("-t","--target_db", default=None, help="URI for the target PostgreSQL database.")
+    parser.add_argument("-s","--source_db", default=DATABASE_URL, help="URI for the source PostgreSQL database.")
+    parser.add_argument("-o","--org_id", required=True, type=int, help="ID of the target organization.")
+    parser.add_argument("-a","--action", default = False,type=bool, help="Whether to update the database.")
+    args = parser.parse_args()
+    transfer_data(target=args.target_db, source=args.source_db,organization_id=args.org_id, action=args.action)
```
Comment (dadiorchen) [inline python/ETL/transfer.py:160]: Can you change this to the pattern as here?
https://github.com/Greenstand/treetracker-airflow-dags/blob/360580afffd51021661c3e88a78f8bae1462ed72/lib/capture_export.py
https://github.com/Greenstand/treetracker-airflow-dags/blob/360580afffd51021661c3e88a78f8bae1462ed72/lib/capture_export_test.py

Here is a simple guide for this test pattern:
https://github.com/Greenstand/treetracker-airflow-dags#option-1-develop-without-installing-airflow

By doing this, we can:
* Make this function platform-agnostic, so it is easy to be reused by: cli, faas, airflow
* Easier to be tested, unit test lib used in the sample code is a good tool for testing and developing, you can easily check your code even hot-refresh your code once you change sometime in your sourcecode.

**Review by dadiorchen — CHANGES_REQUESTED** [review]

**Comment by dadiorchen** [comment]
@Xinyihe123 thank you for the contribution! Above is some suggestion from my side.

File: python/ETL/transfer.py (hunk 1 above)
Comment (Xinyihe123) [reply python/ETL/transfer.py]: Oh sorry for that. Didn't realize that's a public one.

File: python/ETL/transfer.py (hunk 2 above)
Comment (Xinyihe123) [reply python/ETL/transfer.py]: Sure. I'll add the test file later.

**Comment by dadiorchen** [comment]
@Xinyihe123 your code looks good, I will run your script to try to do a migration, will back to you with my result.

**Review by dadiorchen — APPROVED** [review]

**Comment by dadiorchen** [comment]
@Xinyihe123 all looks good, I will try to run your script to do a migration, will back to you with my feedback.

**Comment by dadiorchen** [comment]
@Xinyihe123 I can not run the test, did you run it on your side? This is my command and result:

```
python3 -m unittest discover -s . -p "*_test.py" -v
```


```
  952
  953 ======================================================================
  954 ERROR: test (transfer_test.Test_Transfer)
  955 ----------------------------------------------------------------------
  956 Traceback (most recent call last):
  957   File "/root/treetracker-functions/python/ETL/transfer_test.py", line 20, in test
  958     transfer(dest_conn,src_conn, 11, action = False)
  959 TypeError: 'module' object is not callable
  960
  961 ----------------------------------------------------------------------
  962 Ran 1 test in 0.505s
  963
  964 FAILED (errors=1)
```

**Comment by Xinyihe123** [comment]
> python3 -m unittest discover -s . -p "*_test.py" -v

Yes I can run it on my side. It should be the import format, I've fixed it, it should be able to run on your side now.

**Pull request merged**